hack/.packages

```diff
@@ -65,6 +65,7 @@ k8s.io/kops/pkg/dns
 k8s.io/kops/pkg/edit
 k8s.io/kops/pkg/featureflag
 k8s.io/kops/pkg/flagbuilder
+k8s.io/kops/pkg/formatter
```

Comment:
Another pkg? Can we put it in instancegroups?

Reply:
I was thinking we would start putting all the formatter functionality into this package.  And we've seen that comingling packages doesn't work in go - eventually the circular dependency catches up with you :-)